pydrad/parse/parse.py

```diff
@@ -525,6 +436,102 @@ def grid_edges_right(self) -> u.cm:
         """
         return self.grid_edges[1:]
 
+    @property
+    @u.quantity_input
+    def coordinate(self) -> u.cm:
+        """
+        Coordinate in the field-aligned direction
+        """
+        return self.grid_centers
+
+    @property
+    @u.quantity_input
+    def electron_mass_density(self) -> u.Unit('g cm-3'):
+        # TODO: Account for possible presence of both electron
+        # and ion mass densities. If the electron mass density
+        # is present in this file, it will shift all of the
+        # indices in the .amr file by one.
+        if 'electron_mass_density' in self._amr_data.colnames:
+            return self._amr_data['electron_mass_density']
+        return self.ion_mass_density
+
+    @property
+    @u.quantity_input
+    def ion_mass_density(self) -> u.Unit('g cm-3'):
```

Comment:
**Total** mass density, including all states of all elements (neutrals too!).

Reply:
Ah. Oops!